apps/api/src/domain/models/user-management/contributor/views/contribution-summary.view-model.ts

```diff
@@ -7,14 +11,58 @@ import { NonEmptyString } from '@coscrad/data-types';
  */
 export class ContributionSummary {
     @NonEmptyString({
-        label: 'contributor ID',
-        description: 'ID of person who contributed to the creation or editing of this resource',
+        label: 'contributor IDs',
+        description:
+            'system references identifying who contributed to the creation or editing of this resource',
     })
-    id: string;
+    contributorIds: AggregateId[];
 
     @NonEmptyString({
-        label: 'full name',
-        description: 'the first and last name of the contributor',
+        label: 'statement',
+        description: 'a statement of this contribution to serve as attribution',
```

Comment:
I find this confusing for some reason, wondering if there's another way of saying it.  I think I'm stumbling on the'serve as attribution' part

Reply:
I think we had some discord discussion around this. I'll give it another go.

Reply:
"a written acknowledgement of the contribution"